### PR TITLE
Translate GitHub Desktop to Spanish

### LINKED ISSUE #591: Grammar updates to Spanish translation Intro to GitHub
The Spanish translation of _Intro to GitHub_ is [now live](https://services.github.com/on-demand/intro-to-github/es/) 🎉

While I was deploying this internally, @dpmex4527 🙇 suggested a few updates to the Spanish grammar, which I am copying here so it is publicly visible and we can chip away at it.

/cc: @mberasategi since you submitted the initial translation ❤️ 

screen shot 2017-09-15 at 11 12 31 am
screen shot 2017-09-15 at 11 12 20 am
screen shot 2017-09-15 at 11 12 09 am
screen shot 2017-09-15 at 11 12 03 am
screen shot 2017-09-15 at 11 11 54 am
screen shot 2017-09-15 at 11 11 46 am

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it!
- [ ] Fork the [repository](https://github.com/github/training-kit).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com/on-demand/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository.
- [ ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/on-demand`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!

### PR DESCRIPTION
**Pull request by mberasategi**
## Overview
**TL;DR**
<One to two sentence description of the issue you are encountering or trying to solve.>

Correction of spelling/grammar mistakes of Spanish translation, closes #591 . With the exception of `normbas>nombras` because it was actually meant to be `normas` (rules).

### CODE REVIEW

**Comment by hectorsector** [comment]
@dpmex4527 this PR commits your suggested corrections.

@mberasategi I'll merge this in a day or two if we don't hear from @dpmex4527. I may also be able to get a few other Spanish-speaking folks to chime in.

**Comment by dpmex4527** [comment]
@mberasategi good catch! I meant to suggest `normas` but it seems like I misspelled it. Oops!

**Comment by mberasategi** [comment]
OK @hectorsector and @dpmex4527 , the GitHub Desktop course is now completely translated into Spanish. Some considerations:

- I have not been using GitHub Flow to translate these courses that teach GitHub Flow :persevere: , so this pull request is now a mess. Sorry about that! I promise to create a specific branch for future course translations :innocent: 
- I would say the use of **bold**, 'apostrophes' and `inline-code-style` (even some _italics_ if I recall correctly) to signify buttons, menu options, tabs, form fields etc. is not consistent. This would be a huge and boring task, but there it is. The same with capital letters for git commands like pull request or Pull Request, push and Push, etc. 

Let me know about the translation, and hope students can see it live soon!

**Comment by hectorsector** [comment]
You're on a roll, @mberasategi! 🎉 

I brought in the most recent changes from `master`, and changed the structure around to use improvements we shipped in #593.

@mberasategi @github/services-training could you put some 👀 on this? I can ship with a couple of 👍. It's [staged](https://hectorsector.github.io/training-kit/) for easier review.

**Comment by mberasategi** [comment]
I've checked and links work correctly. I've noticed some typos and
inconsistencies in names while I clicked on everything, will correct those
when I get a moment. Thanks!

El 28 sept. 2017 5:50 a. m., "Hector Alfaro" <notifications@github.com>
escribió:

You're on a roll, @mberasategi <https://github.com/mberasategi>! 🎉

I brought in the most recent changes from master, and changed the structure
around to use improvements we shipped in #593
<https://github.com/github/training-kit/pull/593>.

@mberasategi <https://github.com/mberasategi> @github/services-training
<https://github.com/orgs/github/teams/services-training> could you put some
👀 on this? I can ship with a couple of 👍. It's staged
<https://hectorsector.github.io/training-kit/> for easier review.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/github/training-kit/pull/595#issuecomment-332720141>,
or mute the thread

**Comment by mberasategi** [comment]
Done. Someone else :eyes: for language, style etc.!

**Comment by mberasategi** [comment]
I've just created a gh-pages in my fork to change the config file and make it available to students while it goes live. Is this OK? Should I do it otherwise?

**Comment by hectorsector** [comment]
> I've just created a gh-pages in my fork to change the config file and make it available to students while it goes live.

That should work for you! I'm only merging in what we've got on the master branch!

**Comment by hectorsector** [comment]
Thanks for the submission, @mberasategi! Apologies that it took me so long to merge it in. I was giving folks some time to review it. 🚢ing!

File: _includes/help-gh101-common-es.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Hay un par de razones por las que los builds pueden fallar en esta clase. Si tu
 
 #### El nombre del archivo es incorrecto
 
-El archivo para la clase debería estar en la carpeta `/_pins` y debería es´tar nombrado como `TU-USUARIO`.json. Comprueba para asegurarte de que:
```
Comment (LOve-luST-LOST) [inline _includes/help-gh101-common-es.md:7]: Delete u dumb bitxh

File: index_es.html [hunk 2]
```diff
@@ -13,11 +13,11 @@
     image: images/about/repo.png
     imageAlt: The 'repo' octocat, cartoon of five octocats inside of a metal box
     link: /intro-to-github/es
-  - title: "GitHub 102: GitHub Pages desde GitHub Desktop (en inglés)"
```
Comment (LOve-luST-LOST) [inline index_es.html:16]: Haha
